sdk/communication/communication-messages-rest/src/messagesServiceClient.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TokenCredential, isTokenCredential, KeyCredential, isKeyCredential } from "@azure/core-auth";
+import { ClientOptions } from "@azure-rest/core-client";
+import { parseClientArguments,  createCommunicationAuthPolicy } from "@azure/communication-common";
+import { MessagesServiceClient } from "./generated/src/clientDefinitions";
+import GeneratedAzureCommunicationMessageServiceClient from "./generated/src/messagesServiceClient";
+
+
+/**
+ * Initialize a new instance of `MessagesServiceClient`
+ * @param connectionString - The connectionString or url of your Communication Services resource.
+ * @param options - the parameter for all optional parameters
+ */
+export default function createClient(
+    connectionString: string,
+    options: ClientOptions,
+  ): MessagesServiceClient;
+  
+  /**
+   * Initialize a new instance of `MessagesServiceClient`
+   * @param endpoint - The endpoint of your Communication Services resource.
+   * @param credentialOrOptions The key or token credential.
+   * @param options - the parameter for all optional parameters
+   */
+  export default function createClient(
+    endpoint: string,
+    credentialOrOptions?: KeyCredential | TokenCredential,
```

Comment:
can we rename this to credential and make it required (remove the ?)

Reply:
fixed it.